src/demangle.cc

```diff
@@ -1293,12 +1299,32 @@ static bool ParseTopLevelMangledName(State *state) {
   }
   return false;
 }
+#endif
 
 // The demangler entry point.
 bool Demangle(const char *mangled, char *out, int out_size) {
+#if defined(OS_WINDOWS)
+  // When built with incremental linking, the Windows debugger
+  // library provides a more complicated `Symbol->Name` with the
+  // Incremental Linking Table offset, which looks like
+  // `@ILT+1105(?func@Foo@@SAXH@Z)`. However, the demangler expects
+  // only the mangled symbol, `?func@Foo@@SAXH@Z`. Fortunately, the
+  // mangled symbol is guaranteed not to have parentheses,
+  // so we search for `(` and extract up to `)`.
+  std::string symbol(mangled);
```

Comment:
Hmm I'm confused. I think I've left a comment in stacktrace_windows-inl.h or somewhere, but the comment has disappeared :(

Anyway, I think calling malloc in signal handler isn't safe, so it'd be nicer to avoid using std::string. https://msdn.microsoft.com/en-us/library/aa272921(v=vs.60).aspx

I'm also worried by Windows APIs.